Header header

string group_id

# List of person IDs of the people in this group
string[] members
float32[] confidences